packages/manager/apps/web/client/app/dns-zone/history/dashboard/view/view.html

```diff
@@ -0,0 +1,15 @@
+<oui-modal
+    data-heading="{{:: 'dashboard_history_popup_title' | translate }}"
+    data-primary-action="$ctrl.goBack()"
+    data-primary-label="{{:: 'dashboard_history_close_popup' | translate }}"
+    data-secondary-disabled="true"
+    data-on-dismiss="$ctrl.goBack()"
+>
+    <oui-spinner ng-if="$ctrl.loadingDnsZoneData"></oui-spinner>
+    <div
+        style="white-space: pre-wrap;"
+        ng-if="!$ctrl.loadingDnsZoneData"
+        class="modal-scroll-auto"
+        ng-bind-html="$ctrl.dnsZoneData"
+    ></div>
```

Comment:
What is the data type in $ctrl.dnsZoneData ? Is it text ?
What is the goal to add style and class ?